File: Nodejs/K08_1sub.ts
export function GetMaxValue(v1: number, v2: number, v3: number): number {
    let temp = v1 >= v2 ? v1 : v2;
    temp = temp >= v3 ? temp : v3;
    return temp;
}

export function GetMinValue(v1: number, v2: number, v3: number): number {
    let temp = v1 < v2 ? v1 : v2;
    temp = temp < v3 ? temp : v3;
    return temp;
}

File: Nim/K08_1sub.nim
proc get_max_value*(v1: int, v2: int, v3: int): int =
    var temp = if v1 >= v2: v1 else: v2
    temp = if temp >= v3: temp else: v3
    return temp

proc get_min_value*(v1: int, v2: int, v3: int): int =
    var temp = if v1 < v2: v1 else: v2
    temp = if temp < v3: temp else: v3
    return temp